examples/cell_tracking.py

```diff
@@ -0,0 +1,174 @@
+        print(f"CSV file has fewer than 4 columns: {len(df.columns)}")
+        return None
+
+    except Exception as e:
+        print(f"Error loading tracks file: {e}")
+        return None
+
+tracks_data = load_tracks_data(Path(data_dir).parent, filename=Path(data_dir).name)
+
+# Add tracks layer with colors
+if tracks_data is not None:
+    print(f"Adding {len(np.unique(tracks_data[:, 0]))} tracks to napari...")
+
+    # Create track properties for coloring
+    track_ids = np.unique(tracks_data[:, 0])
+
+
+    # Detect splitting tracks (tracks that share the same parent or split from one another)
+    # Logic: identify tracks that start at the same time and in close position (less than 10 pixels apart)
+    splitting_tracks = set()
+
+    for track_id in track_ids:
+        track_points = tracks_data[tracks_data[:, 0] == track_id]
+        start_time = track_points[:, 1].min()
+        start_pos = track_points[track_points[:, 1] == start_time][0, 2:4]  # [y, x]
+
+        # Check if other tracks start at same time and nearby position
+        for other_track_id in track_ids:
+            if other_track_id != track_id:
+                other_track_points = tracks_data[tracks_data[:, 0] == other_track_id]
+                other_start_time = other_track_points[:, 1].min()
+
+                if other_start_time == start_time:
+                    other_start_pos = other_track_points[other_track_points[:, 1] == other_start_time][0, 2:4]
+                    # If they start within 10 pixels of each other, consider them splitting
+                    distance = np.sqrt(np.sum((start_pos - other_start_pos)**2))
+                    if distance < 10:
+                        splitting_tracks.add(track_id)
+                        splitting_tracks.add(other_track_id)
+
+    # Create the arrays using the populated splitting_tracks
+    track_id_per_point = tracks_data[:, 0].astype(int)
+    is_splitting_per_point = np.array([int(tid) in splitting_tracks for tid in track_id_per_point], dtype=float)
+
+    # Create features dictionary
+    features = pd.DataFrame({
+    'track_id': track_id_per_point,
+    'is_splitting': is_splitting_per_point
+    })
+
+    # Add tracks layer
+    tracks_layer = viewer.add_tracks(
+        tracks_data,
+        features=features,
+        name='Cell Tracks',
+        tail_length=10,
+        tail_width=2,
+        head_length=0,
+        colormap='viridis',
+        color_by='is_splitting',
+        blending='additive'
+    )
+# Add cell lebels
+# Add cell lebels
+label_url = 'https://zenodo.org/records/15852284/files/masks_pred.npz?download=1'
+label_path = pooch.retrieve(url=label_url,
+                            fname="masks_pred.npz",
+                            known_hash=None,
+                            path=pooch.os_cache("cell_labels")
+                            )
+
+label_data = np.load(label_path)
+labels = label_data["masks"]
+
+viewer.add_labels(labels, name="Predicted Masks")
+napari.run()
```

Comment:
```suggestion

if __name__ == '__main__':
    napari.run()
```

Gah should have caught this earlier, napari.run() is only needed for scripts, so we want to ensure things work for people using jupyter or ipython. See: https://napari.org/stable/developers/contributing/documentation/index.html#adding-examples-to-the-gallery

Reply:
Thanks! I added the main block and I hope this fixes the issues.

Reply:
@Olusesan I don't see the main block here. Perhaps you missed pushing it?